src/Sulu/Bundle/SecurityBundle/Resources/config/doctrine/AccessControl.orm.xml

```diff
@@ -13,7 +13,7 @@
         </id>
 
         <field name="permissions" type="smallint" column="permissions"/>
-        <field name="entityId" type="integer" column="entityId"/>
+        <field name="entityId" type="string" column="entityId" length="191"/>
```

Comment:
Would set the length to `32`. This is in our cases never longer as a uuid we should not use more space as needed to not get to some kind of rowmaxsize in some databases.

Reply:
sorry uuids are `36`